kimchi/src/tests/range_check.rs

```diff
@@ -645,7 +644,6 @@ fn verify_range_check0_v1_test_lookups() {
             TestFramework::<Vesta>::default()
                 .gates(index.cs.gates.clone())
                 .witness(witness.clone())
-                .lookup_tables(vec![range_check::gadget::lookup_table()])
```

Comment:
Oh, interesting. Are the lookup constraints still working without this?  Sounds like I've misunderstood something!

So it automatically knows to include the range-check lookup table without needing to specify it here?  And, by doing it here I was including an extra copy?

Reply:
That's right, yea. The constraint system generator already associates gates with particular lookup tables

Reply:
That's great.  Is it something new or has it always done that?

Reply:
It's always done that!